rosbag2/include/rosbag2/storage_options.hpp

```diff
@@ -25,6 +25,12 @@ struct StorageOptions
 public:
   std::string uri;
   std::string storage_id;
+
+  /**
+   * The maximum size a bagfile can be, in bytes, before it is split.
+   * A value of 0 indicates that bagfile splitting will not be used.
+   */
+  uint64_t max_bagfile_size;
```

Comment:
This needs a visibility macro for windows.
```
09:57:06      Creating library C:/J/workspace/ci_windows/ws/build/rosbag2/Release/rosbag2.lib and object C:/J/workspace/ci_windows/ws/build/rosbag2/Release/rosbag2.exp
09:57:06 writer.obj : error LNK2019: unresolved external symbol "unsigned __int64 const rosbag2_storage::storage_interfaces::MAX_BAGFILE_SIZE_NO_SPLIT" (?MAX_BAGFILE_SIZE_NO_SPLIT@storage_interfaces@rosbag2_storage@@3_KB) referenced in function "public: __cdecl rosbag2::Writer::Writer(class std::unique_ptr<class rosbag2_storage::StorageFactoryInterface,struct std::default_delete<class rosbag2_storage::StorageFactoryInterface> >,class std::shared_ptr<class rosbag2::SerializationFormatConverterFactoryInterface>,class std::unique_ptr<class rosbag2_storage::MetadataIo,struct std::default_delete<class rosbag2_storage::MetadataIo> >)" (??0Writer@rosbag2@@QEAA@V?$unique_ptr@VStorageFactoryInterface@rosbag2_storage@@U?$default_delete@VStorageFactoryInterface@rosbag2_storage@@@std@@@std@@V?$shared_ptr@VSerializationFormatConverterFactoryInterface@rosbag2@@@3@V?$unique_ptr@VMetadataIo@rosbag2_storage@@U?$default_delete@VMetadataIo@rosbag2_storage@@@std@@@3@@Z) [C:\J\workspace\ci_windows\ws\build\rosbag2\rosbag2.vcxproj]
09:57:06 C:\J\workspace\ci_windows\ws\build\rosbag2\Release\rosbag2.dll : fatal error LNK1120: 1 unresolved externals [C:\J\workspace\ci_windows\ws\build\rosbag2\rosbag2.vcxproj]
```
See Windows CI for details: https://ci.ros2.org/job/ci_windows/8395/console

Reply:
Thanks, I think I found the issue. Compiling and testing locally, will push change soon